pkg/download/download_writer_test.go

```diff
@@ -200,6 +255,10 @@ func TestWriteToDisk(t *testing.T) {
 				t.Errorf("WriteToDisk(): expected manifest %v was not created", expectedManifest)
 			}
 
+			actualManifestContent, err := afero.ReadFile(tt.args.fs, expectedManifest)
+			require.NoError(t, err)
+			assert.Equal(t, tt.wantManifestContent, string(actualManifestContent),
+				"WriteToDisk(): manifest content was expected to be:\n%v\nbut actually was:\n%v", tt.wantManifestContent, string(actualManifestContent))
```

Comment:
Nit: I don't think `WriteToDisk():` serves any purpose here (or anywhere in this test)
```suggestion
				"manifest content was expected to be:\n%v\nbut actually was:\n%v", tt.wantManifestContent, string(actualManifestContent))
```